org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/XMLModel.java

```diff
@@ -83,4 +121,17 @@ public static boolean isXMLModel(DOMNode node) {
 		return node.isProcessingInstruction() && XML_MODEL_PI.equals(((ProcessingInstruction) node).getTarget());
 	}
 
+	public DOMRange getHrefNode() {

```

Comment:
Could we have javadoc that explains that this method returns the range for the href value?

Reply:
@xorye fixed